commands/playwright-keplr.js

```diff
@@ -366,4 +335,8 @@ module.exports = {
       );
     }
   },
+  async waitForURLLoad(url, page = keplrWindow) {
+    await page.waitForURL(url);
+    await module.exports.waitUntilStable(page);
+  },
```

Comment:
Why do we need this method?

Reply:
mentioned as point no.3 in the descripiton. whenever we press the change wallet button, it redirects to the page it was previously on. this function is being used there to wait for that redirect